packages/vite/src/node/plugins/css.ts

```diff
@@ -1796,8 +1797,8 @@ function createViteLessPlugin(
         this.resolvers = resolvers
         this.alias = alias
       }
-      override supports() {
-        return true
+      override supports(filename: string) {
+        return !urlRE.test(filename)
```

Comment:
I think technically it's a bug in `rebaseUrls` as scss uses that too, but scss has it's own logic that handles URLs before it allows Vite to, so it doesn't hit this bug. It that case, I'm fine fixing this here for less for now as it's a fast-path.

I think we can use the `isExternalUrl` function from `util.ts` here.

Reply:
PS: I'm fine with not adding a test for this.

Reply:
Even if `rebaseUrls` supports parsing external url, we still need to let `less.UrlFileManager` handle external CSS. But I can add an extra fix to `rebaseUrls`.

I've committed the code which uses `isExternalUrl`.